src/spikeinterface/sorters/external/kilosort.py

```diff
@@ -188,27 +252,31 @@ def _get_specific_options(cls, ops, params) -> dict:
         # the following options can improve/deteriorate results.
         # when multiple values are provided for an option, the first two are beginning and ending anneal values,
         # the third is the value used in the final pass.
-        ops["Th"] = [4.0, 10.0, 10.0]  # threshold for detecting spikes on template-filtered data ([6 12 12])
-        ops["lam"] = [5.0, 5.0, 5.0]  # large means amplitudes are forced around the mean ([10 30 30])
-        ops["nannealpasses"] = 4.0  # should be less than nfullpasses (4)
-        ops["momentum"] = [1 / 20, 1 / 400]  # start with high momentum and anneal (1./[20 1000])
-        ops["shuffle_clusters"] = 1.0  # allow merges and splits during optimization (1)
-        ops["mergeT"] = 0.1  # upper threshold for merging (.1)
-        ops["splitT"] = 0.1  # lower threshold for splitting (.1)
-
-        ops["initialize"] = "fromData"  # 'fromData' or 'no'
+        ops["Th"] = params["Th"]  # threshold for detecting spikes on template-filtered data ([6 12 12])
+        ops["lam"] = params["lam"]  # large means amplitudes are forced around the mean ([10 30 30])
+        ops["nannealpasses"] = params["nannealpasses"]  # should be less than nfullpasses (4)
+        assert ops["nannealpasses"] < ops["nfullpasses"]
```

Comment:
```suggestion
        assert ops["nannealpasses"] < ops["nfullpasses"], f"{nannealpasses=} should be less than {nfullpasses=}"
```
or you could make it smoother if you wanted to write out a proper error message!  But we prefer when possible to show the end-user what they are doing wrong :)